scripts/host-build-all.sh

```diff
@@ -5,17 +5,59 @@
 # fail on any errors
 set -e
 
-cd tools
-cd testbench
+print_usage()
+{
+    cat <<EOFUSAGE
+usage: $0 [-f]
+       -f Build testbench with compiler provided by fuzzer
+          (default path: /home/sof/work/AFL/afl-gcc)
```

Comment:
We should make this path relative to $HOME

Reply:
Agree. I'll change that.

Reply:
Agree. Made the changes. Updated commit description too, please take a look at it.